modules/story-budget/story-budget.php

```diff
@@ -188,15 +188,20 @@ function register_term_columns() {
 	 */
 	function handle_form_date_range_change() {
 		
-		if ( !isset( $_POST['ef-story-budget-range-submit'], $_POST['ef-story-budget-number-days'], $_POST['ef-story-budget-start-date'] ) )
+		if ( !isset( $_POST['ef-story-budget-number-days'], $_POST['ef-story-budget-start-date'] ) &&
+		     ( !isset( $_POST['ef-story-budget-range-submit'] ) || !isset( $_POST['ef-story-budget-today-submit'] ) ) )
 			return;
 			
 		if ( !wp_verify_nonce( $_POST['nonce'], 'change-date' ) )
 			wp_die( $this->module->messages['nonce-failed'] );
 		
 		$current_user = wp_get_current_user();
 		$user_filters = $this->get_user_meta( $current_user->ID, self::usermeta_key_prefix . 'filters', true );
-		$user_filters['start_date'] = date( 'Y-m-d', strtotime( $_POST['ef-story-budget-start-date'] ) );
+		if ( isset( $_POST['ef-story-budget-today-submit'] ) ) {
+			$user_filters['start_date'] = date( 'Y-m-d' );
+		} else {
+			$user_filters['start_date'] = date( 'Y-m-d', strtotime( $_POST['ef-story-budget-start-date'] ) );
```

Comment:
For consistent behavior:

```suggestion
			$user_filters['start_date'] = date_i18n( 'Y-m-d', strtotime( $_POST['ef-story-budget-start-date'] ) );
```

Reply:
Ah interesting, this will need a patch similar to #546. `$_POST['ef-story-budget-start-date']` is not going to be parsed correctly by `strtotime` since it's value is `Y-m-d`

Either way, `date_i18n` seems more consistent, I can create a new PR for the localization issue